admin_guide/native_container_routing.adoc

```diff
@@ -10,58 +10,77 @@
 toc::[]
 
 
-== Introduction
-This document describes how one could setup container networking using existing switches/routers and using
-the kernel networking stack in Linux. The setup requires that the network administrator or some script
-modifies the router[s] as new nodes are added to the cluster. The document will describe the steps to
-use a Linux server as a simple router. The steps could then be adapted to any particular router.
+== Overview
+This topic describes how to set up container networking using existing switches
+and routers and the kernel networking stack in Linux. The setup requires that
+the network administrator or a script modifies the router or routers when new
+nodes are added to the cluster.
+
+[NOTE]
+The procedures outlined in this topic can be adapted to any type of router.
 
 
 == Network Layout
-The diagram below shows the setup used in the document. It has one Linux node with two network interface
-cards serving as a router, two switches and three nodes connected to these switches.
+The following diagram shows the container networking setup described in this
+topic. It uses one Linux node with two network interface cards serving as a
+router, two switches, and three nodes connected to these switches.
 
 image::network_diagram.png["Network Diagram"]
 
 == Network Overview
+The following describes a general network setup:
+
 * 11.11.0.0/16 is the container network.
-* 11.11.x.0/24 subnet is reserved for each node and assigned to the docker linux bridge.
-* Each node has a route to the router for reaching anything 11.11.0.0/16 except the local subnet.
-* Router has routes for each node, so it can direct to the right node.
-* Nodes don't need any changes when new nodes are added unless the network topology is modified.
+* The 11.11.x.0/24 subnet is reserved for each node and assigned to the docker linux bridge.
+* Each node has a route to the router for reaching anything in the 11.11.0.0/16 range, except the local subnet.
+* The router has routes for each node, so it can be directed to the right node.
+* Existing nodes do not need any changes when new nodes are added, unless the network topology is modified.
 * IP forwarding is enabled on each node.
 
 == Node setup
-* Assign an unused 11.11.x.0/24 subnet ip address to the linux bridge on the node.
+
+. Assign an unused 11.11.x.0/24 subnet IP address to the Linux bridge on the node:
++
 ----
-brctl addbr lbr0
-ip addr add 11.11.1.1/24 dev lbr0
-ip link set dev lbr0 up
+$ brctl addbr lbr0
+$ ip addr add 11.11.1.1/24 dev lbr0
+$ ip link set dev lbr0 up
 ----
-* Modify docker startup script to use the new bridge (/etc/sysconfig/docker on Fedora).
+
+. Modify the docker startup script to use the new bridge. By default, the
+startup script is found in the `/etc/sysconfig/docker` directory:
++
 ----
```

Comment:
On my system `/etc/sysconfig/docker` is a file.  I believe it is the script referenced in the first sentence.